livecheck/settings.py

```diff
@@ -101,6 +113,9 @@ def gather_settings(search_dir: str) -> LivecheckSettings:
                 semver[catpkg] = settings_parsed['semver']
             if 'jetbrains' in settings_parsed:
                 jetbrains_packages[catpkg] = settings_parsed['jetbrains']
+            if 'keep_old' in settings_parsed:
+                keep_old[catpkg] = to_bool(settings_parsed['keep_old'])
```

Comment:
I do not want the `to_bool` function. `livecheck.json` should only use real boolean values for that. There's no reason not to.